Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM registry.svc.ci.openshift.org/ocp/builder:golang-1.12 AS builder
+FROM registry.svc.ci.openshift.org/ocp/builder:golang-1.13 AS builder
```

Comment:
I don't think changes not matching `go.{mod,sum} vendor/*` belong to a bump commit

Reply:
will be fixed in next push